<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportSummaryOnServicesDialog</class>
 <widget class="QDialog" name="ReportSummaryOnServicesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblMedicalAidUnit">
     <property name="text">
      <string>Единица измерения</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbMedicalAidUnit"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chkClass">
     <property name="text">
      <string>Класс</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QComboBox" name="cmbClass">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Статус</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Диагностика</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Лечение</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Прочие мероприятия</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Назначено</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblActionTypeGroup">
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="CActionTypeComboBoxMCL" name="cmbActionTypeGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblPerson">
     <property name="text">
      <string>Исполнитель</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="CPersonComboBoxEx" name="cmbPerson"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QCheckBox" name="chkConfirmation">
     <property name="text">
      <string>Подтверждение оплаты</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblConfirmationType">
     <property name="text">
      <string>Тип подтверждения:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="4">
    <widget class="QComboBox" name="cmbConfirmationType">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>не выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>оплачено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>отказано</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblBegDateConfirmation">
     <property name="text">
      <string>Начало периода подтверждения:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="CDateEdit" name="edtConfirmationBegDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QComboBox" name="cmbConfirmationPeriodType">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>по дате формирования счета</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по дате подтверждения оплаты</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>165</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblEndDateConfirmation">
     <property name="text">
      <string>Окончание периода подтверждения:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="CDateEdit" name="edtConfirmationEndDate">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>222</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="14" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="3" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="4">
    <widget class="QCheckBox" name="chkVAT">
     <property name="text">
      <string>Без учёта НДС</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CPersonComboBoxEx</class>
   <extends>QComboBox</extends>
   <header>Orgs/PersonComboBoxEx.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CActionTypeComboBoxMCL</class>
   <extends>QComboBox</extends>
   <header>Events/ActionTypeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbMedicalAidUnit</tabstop>
  <tabstop>chkClass</tabstop>
  <tabstop>cmbClass</tabstop>
  <tabstop>cmbActionTypeGroup</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbPerson</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>chkConfirmation</tabstop>
  <tabstop>cmbConfirmationType</tabstop>
  <tabstop>edtConfirmationBegDate</tabstop>
  <tabstop>cmbConfirmationPeriodType</tabstop>
  <tabstop>edtConfirmationEndDate</tabstop>
  <tabstop>chkVAT</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportSummaryOnServicesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportSummaryOnServicesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkClass</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbClass</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkClass</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbActionTypeGroup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkConfirmation</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbConfirmationPeriodType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkConfirmation</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbConfirmationType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkConfirmation</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtConfirmationBegDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkConfirmation</sender>
   <signal>toggled(bool)</signal>
   <receiver>edtConfirmationEndDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
